scripts/utilities.js

```diff
@@ -25,6 +25,10 @@ function createElement(type = "add", websiteUrl = "https://vibinex.com") {
 			imgUrl = "https://cdn-icons-png.flaticon.com/512/1243/1243911.png?w=740&t=st=1680153899~exp=1680154499~hmac=be129e6a5a3dd4b9a362138086907f3330050f0a300473c5ed0e7e9541ece2de"
 			bannerMessage = "Something went wrong";
 			break;
+		case "login":
+			loadingIconID = "vibinexLoginIcon";
+			imgUrl = "https://cdn-icons-png.flaticon.com/512/6980/6980070.png?filename=user_6980070.png&fd=1";
+			bannerMessage = "Please Login to Vibinex"
```

Comment:
You will need to create a separate UI for this case since this particular case needs to be a lot more prominent than any other.

You can choose to use the rest of this structure and create a function for the creation of the HTML elements. (The existing code is not written in the most maintainable manner. But fixing it is obviously optional.)